README.md

```diff
@@ -43,10 +43,16 @@ done
 There is a manifest to deploy DVO via OLM artifacts.  This assumes that OLM is already running in the cluster.  To deploy via OLM:
 
 * Generate the deployment YAML from the openshift template
-* Deploy the generated YAML
+* Deploy one of the two following YAML templates (not both!):
 
 ```
-oc process --local NAMESPACE_IGNORE_PATTERN='openshift.*|kube-.+' -f deploy/openshift/deployment-validation-operator-olm.yaml | oc create -f -
+# deploy this if you DO NOT want OLM to automatically upgrade DVO
+oc process --local NAMESPACE_IGNORE_PATTERN='openshift.*|kube-.+' -f deploy/openshift/deployment-validation-operator-olm-without-polling.yaml | oc create -f -
+```
+
+```
+# otherwise, deploy this if you DO want OLM to automatically upgrade DVO
+oc process --local NAMESPACE_IGNORE_PATTERN='openshift.*|kube-.+' -f deploy/openshift/deployment-validation-operator-olm-with-polling.yaml | oc create -f -
```

Comment:
I would add another bit here saying that by default OLM will look to update the operator every X minutes/days.  To change the interval do this: ...

Reply:
Now says:

```
# otherwise, deploy this if you DO want OLM to automatically upgrade DVO
# (set REGISTRY_POLLING_INTERVAL to be shorter to have OLM check for new DVO versions more frequently if desired; e.g. '45m')
oc process --local \
NAMESPACE_IGNORE_PATTERN='openshift.*|kube-.+' \
REGISTRY_POLLING_INTERVAL='24h' \
-f deploy/openshift/deployment-validation-operator-olm-with-polling.yaml \
| oc create